expressEngine/cms/user/error.js

```diff
@@ -7,4 +7,7 @@ export default {
   loginError: 'login failed please signUp first',
   passwordError: 'incorrect password',
   tokenError: 'token is not generated',
+  unableToUpdate: 'User not found to update',
```

Comment:
@adilsuccessive please change unableToUpdate, if data to update is already there in the database then nModified will give 0 and unableToUpdate message will be displayed, but this message is related to user not found, you can set generic value 'unable to update'.

Reply:
User not found to update is displayed if no records found to update during the search.